app/models/visualization.rb

```diff
@@ -0,0 +1,4 @@
+class Visualization < ApplicationRecord
+  has_many :variables, inverse_of: :visualization
+  accepts_nested_attributes_for :variables
```

Comment:
Could you explain why we use the `inverse_of:` parameter and `accepts_nested_attributes_for` method?

Reply:
The `inverse_of:` parameter basically tells Rails that two models have the same relationship but in different directions. So, since Visualization hasmany Variables and Variable belongsto Visualization, it's saying that the :visualization relation on Variable is the inverse_of the :variables relation on Visualization (basically, an aspect of making models properly associated)

The `accepts_nested_attributes_for` lets you enter and save fields for a Visualization's Variables when creating that Visualization (basically, it's accepting the parameters for Variables, which are a nested model of Visualization since Visualization hasmany Variables)

Reply:
Okay thanks for explaining that, this looks great.